### ISSUE TITLE
beta readme change

### ISSUE THREAD

**Issue by hammy4785** [cviebrock/eloquent-sluggable#155]
Can you update the readme section for model when its null you need Cocur/Slugify

**Comment by cviebrock** [comment]
I'm not sure I follow you.  The docs at https://github.com/cviebrock/eloquent-sluggable#method say that if you don't provide your own method, then we use cocur/slugify.  And `composer.json` requires it now, so you shouldn't need to manually install it or anything.

What did I miss?

**Comment by hammy4785** [comment]
To start the issue wasn’t in README.md so that’s my bad its in config
> sluggable.php (within the app and within elloquentsluggable folders)
however please note that when I run:



composer require cviebrock/eloquent-sluggable ">=3.0.0-beta”



The only item that was setup was eloquent not cocur. The config file
states the below which was what I looked at when testing to debug.



/**
 * If left to "null", then use Laravel's built-in Str::slug() method to
 * generate the slug (with the separator defined below).





-- 



Sincerely,

Mike Oltmans



Director of Online Development

The Maritime Executive

Work: 954 848 9955

Cell: 954 249 4908



*From:* Colin Viebrock [mailto:notifications@github.com]
*Sent:* Wednesday, June 24, 2015 3:14 PM
*To:* cviebrock/eloquent-sluggable
*Cc:* hammy4785
*Subject:* Re: [eloquent-sluggable] beta readme change (#155)



I'm not sure I follow you. The docs at
https://github.com/cviebrock/eloquent-sluggable#method say that if you
don't provide your own method, then we use cocur/slugify. And composer.json
requires it now, so you shouldn't need to manually install it or anything.

What did I miss?

—
Reply to this email directly or view it on GitHub
<https://github.com/cviebrock/eloquent-sluggable/issues/155#issuecomment-114984207>
.

**Comment by hammy4785** [comment]
So are they supposed to be like that?







-- 



Sincerely,

Mike Oltmans



Director of Online Development

The Maritime Executive

Work: 954 848 9955

Cell: 954 249 4908



*From:* Colin Viebrock [mailto:notifications@github.com]
*Sent:* Wednesday, June 24, 2015 3:38 PM
*To:* cviebrock/eloquent-sluggable
*Cc:* hammy4785
*Subject:* Re: [eloquent-sluggable] beta readme change (#155)



Closed #155 <https://github.com/cviebrock/eloquent-sluggable/issues/155>
via dc0a239
<https://github.com/cviebrock/eloquent-sluggable/commit/dc0a2399dcbe4c8d0ea3f7e21e039f7340b45ffe>
.

—
Reply to this email directly or view it on GitHub
<https://github.com/cviebrock/eloquent-sluggable/issues/155#event-339315697>
.

**Comment by cviebrock** [comment]
Just tried it with a fresh Laravel install:

```shell
$ laravel new l5test
Crafting application...
Generating optimized class loader
Compiling common classes
Application key [suMCKXLeuDuaJwEIxYiZYa1vJ1i4rYVH] set successfully.
Application ready! Build something amazing.

$ cd l5test

$ composer require "cviebrock/eloquent-sluggable:>=3.0.0-beta"
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Installing cocur/slugify (v1.1)
    Downloading: 100%

  - Installing cviebrock/eloquent-sluggable (3.0.0-beta)
    Downloading: 100%

Writing lock file
Generating autoload files
Generating optimized class loader
Compiling common classes
```

**Comment by hammy4785** [comment]
Awesome. Glad it works correctly.







-- 



Sincerely,

Mike Oltmans



Director of Online Development

The Maritime Executive

Work: 954 848 9955

Cell: 954 249 4908



*From:* Colin Viebrock [mailto:notifications@github.com]
*Sent:* Wednesday, June 24, 2015 3:48 PM
*To:* cviebrock/eloquent-sluggable
*Cc:* hammy4785
*Subject:* Re: [eloquent-sluggable] beta readme change (#155)



Just tried it with a fresh Laravel install:

$ laravel new l5test

Crafting application...

Generating optimized class loader

Compiling common classes

Application key [suMCKXLeuDuaJwEIxYiZYa1vJ1i4rYVH] set successfully.

Application ready! Build something amazing.



$ cd l5test



$ composer require "cviebrock/eloquent-sluggable:>=3.0.0-beta"

./composer.json has been updated

Loading composer repositories with package information

Updating dependencies (including require-dev)

  - Installing cocur/slugify (v1.1)

    Downloading: 100%



  - Installing cviebrock/eloquent-sluggable (3.0.0-beta)

    Downloading: 100%



Writing lock file

Generating autoload files

Generating optimized class loader

Compiling common classes

—
Reply to this email directly or view it on GitHub
<https://github.com/cviebrock/eloquent-sluggable/issues/155#issuecomment-114991425>
.

**Comment by hammy4785** [comment]
I just tested as well and I get the same result when copy pasting your
instructions from github so definitely working now. Thanks for taking the
time to get back to me!







-- 



Sincerely,

Mike Oltmans



Director of Online Development

The Maritime Executive

Work: 954 848 9955

Cell: 954 249 4908



*From:* Colin Viebrock [mailto:notifications@github.com]
*Sent:* Wednesday, June 24, 2015 3:48 PM
*To:* cviebrock/eloquent-sluggable
*Cc:* hammy4785
*Subject:* Re: [eloquent-sluggable] beta readme change (#155)



Just tried it with a fresh Laravel install:

$ laravel new l5test

Crafting application...

Generating optimized class loader

Compiling common classes

Application key [suMCKXLeuDuaJwEIxYiZYa1vJ1i4rYVH] set successfully.

Application ready! Build something amazing.



$ cd l5test



$ composer require "cviebrock/eloquent-sluggable:>=3.0.0-beta"

./composer.json has been updated

Loading composer repositories with package information

Updating dependencies (including require-dev)

  - Installing cocur/slugify (v1.1)

    Downloading: 100%



  - Installing cviebrock/eloquent-sluggable (3.0.0-beta)

    Downloading: 100%



Writing lock file

Generating autoload files

Generating optimized class loader

Compiling common classes

—
Reply to this email directly or view it on GitHub
<https://github.com/cviebrock/eloquent-sluggable/issues/155#issuecomment-114991425>
.
